src/types/nvi.types.ts

```diff
@@ -27,16 +27,22 @@ export interface NviCandidate {
   approvals: NviCandidateApproval[];
 }
 
-interface NviCandidateAggregations {
-  'approvalStatus.rejected': {
-    docCount: number;
-  };
-  'approvalStatus.approved': {
-    docCount: number;
-  };
-  'approvalStatus.pending': {
-    docCount: number;
-  };
+interface AggregationCount {
+  docCount: number;
+}
+
+export interface NviCandidateAggregations {
+  approved: AggregationCount;
+  // approvedCollaboration: AggregationCount;
+  // assigned: AggregationCount;
+  // assignedCollaboration: AggregationCount;
+  pending: AggregationCount;
+  // pendingCollaboration: AggregationCount;
+  rejected: AggregationCount;
+  // rejectedCollaboration: AggregationCount;
+  // assignments: AggregationCount;
+  // completed: AggregationCount;
+  // totalCount: AggregationCount;
 }
```

Comment:
kommenterer ut mange av disse da de relativt snart skal brukes, men om de er med nå mister vi kontroll over typen `StatusFilterMapping` som sikrer at vi har mapping mot søket for alle statuser